hwy/ops/shared-inl.h

```diff
@@ -480,6 +480,8 @@ using BlockDFromD =
 #define HWY_IF_NOT_T_SIZE_D(D, bytes) HWY_IF_NOT_T_SIZE(TFromD<D>, bytes)
 #define HWY_IF_T_SIZE_ONE_OF_D(D, bit_array) \
   HWY_IF_T_SIZE_ONE_OF(TFromD<D>, bit_array)
+#define HWY_IF_T_SIZE_LT_D(D, bytes) HWY_IF_T_SIZE_LT(TFromD<D>, bytes)
```

Comment:
Here also _LE?

Reply:
I have made the change from LT to LE here.